http/getAxiosConfig.ts

```diff
@@ -4,6 +4,26 @@ import { getHubSpotApiOrigin } from '../lib/urls';
 import { AxiosConfigOptions } from '../types/Http';
 import { CLIConfig } from '../types/Config';
 import { AxiosRequestConfig } from 'axios';
+import https from 'https';
+import http from 'https';
```

Comment:
should this be `http`?

Reply:
It sure should be, great catch.

Reply:
Commit to address the issue and update tests to catch it https://github.com/HubSpot/hubspot-local-dev-lib/pull/139/commits/e1e396dee2a37247a204f04c3cd373df22ad9df0